### PR TITLE
Completed Protected Index

### PR DESCRIPTION
**Pull request by scharlez**

### CODE REVIEW

File: app/javascript/components/App.js [hunk 1]
```diff
@@ -76,23 +76,44 @@ class App extends Component {
...
         <Switch>
           <Route exact path="/" component={Home} />
-          <Route path="/tripindex" component={TripIndex} />
-          <Route path="/tripshow/:id" render={(props) =>{
+          <Route
+							path="/tripindex"
+							render={(props) => (
+								<TripIndex {...props} trips={this.state.trips} />
+							)}
+						/>
```
Comment (elysemontano) [inline app/javascript/components/App.js]: The indentation looks a little off here.

File: app/javascript/components/App.js [hunk 2]
```diff
@@ -76,23 +76,44 @@ class App extends Component {
...
+              render={(props) => {
+								let myTrips = this.state.trips.filter(
+									(trip) => trip.user_id === current_user.id)
+								return (
+									<ProtectedTripIndex
+										trips={myListings}
+										deleteTrip={this.deleteTrip}
+									/>
+								)
+						}}/>
```
Comment (elysemontano) [inline app/javascript/components/App.js]: The indentation looks a little off here as well.

File: app/javascript/components/pages/TripIndex.js [hunk 3]
```diff
@@ -1,9 +1,57 @@
...
+										</CardTitle>
+									</Card>
+								</Col>
+							);
+						})}
+				</Row>
+			</>
+		);
+	}
+}
```
Comment (elysemontano) [inline app/javascript/components/pages/TripIndex.js]: Check indentation

**Review by elysemontano — CHANGES_REQUESTED** [review]
Please fix indentation where I left comments.

**Comment by scharlez** [comment]
Please review revisions.

**Review by elysemontano — CHANGES_REQUESTED** [review]
Indentation looks a little off still on App.js and TripIndex.  Check that the page has tab spacing set to 2 or 4 then format the page.

**Review by triciasykes — CHANGES_REQUESTED** [review]
Trip Index is fine,  but now App.js is off

**Review by triciasykes — APPROVED** [review]

**Review by elysemontano — APPROVED** [review]

**Pull request merged**